lib/pxproxy.js

```diff
@@ -120,12 +120,22 @@ function parseBody(req) {
                     limit: '1mb',
                     encoding: contentType.parse(req).parameters.charset
                   }, function (err, string) {
-                    req.body = string.toString();
+                    req.rawBody = string.toString();
                     resolve();
                 })
             }
             else {
-                resolve();
+                if (Object.keys(req.body).length >= 0) {
```

Comment:
is there a chance that `req.body` is null here? if so, this will crash

Reply:
it cant be null here, because the if states `if (!req.body)` so if its null it will go into the if condition :)

Reply:
even if `req,body` is a string, this condition will evaluate to `true`. instead, you can use `typeof req.bosy === 'object'`